app/controllers/package_set.js

```diff
@@ -47,14 +55,19 @@ export default Controller.extend({
   ),
   packageUnavailableInSet: computed(
     "model",
+    "model.isSet",
+    "allPackages.@each.isAvailable",
     "allPackages.@each.availableQuantity",
     function() {
       if (!this.get("model")) {
         return true;
       }
-      let quantity = this.get("allPackages").any(
-        pkg => pkg.get("availableQuantity") == 0
-      );
+      let quantity = this.get("allPackages").any(pkg => {
+        return (
+          pkg.get("availableQuantity") == 0 ||
+          pkg.get("allowWebPublish") == false
+        );
+      });
```

Comment:
Can we add it like below?

```suggestion
      let quantity = this.get("allPackages").any(pkg => 
          pkg.get("availableQuantity") == 0 || pkg.get("allowWebPublish") == false
      });
```